utils.py

```diff
@@ -431,13 +442,31 @@ def capitalize_string(input_string):
 
 
 def get_hex_color_for_category(category):
+    """Given the category, it returns the color associated with the category,
+    if the category is present in the app constants else given a default color.
+
+    Args:
+      category : Category to get color
+
+    Returns:
+      Color assigned to that category.
+    """
     return (
         constants.CATEGORIES_TO_COLORS[category]
         if category in constants.CATEGORIES_TO_COLORS
         else constants.DEFAULT_COLOR)
 
 
 def get_thumbnail_icon_url_for_category(category):
+    """Given the category, it returns the associated thumbnail icon, if the
+    category is present in the app constants else given a default thumbnail.
+
```

Comment:
Change given --> returns

Reply:
Please address this comment. Else everything is fine.

Reply:
Done :+1: